File: Python/BubbleSort.py
def bubble_sort(array):
    for i in range(len(array)):
        for j in range(i+1, len(array)):
            if array[i] > array[j]:
                temp = array[i]
                array[i] = array[j]
                array[j] = temp
    
    return array

x = [1, 2, 34, 782, 4, 5, 9, -8, -9]
print(x)
print(bubble_sort(x))

File: Javascript/BubbleSort.js
function BubbleSort(array) {
    for (let i = 0; i < array.length; i++) {
        for (let j = i+1; j < array.length; j++) {
            if (array[i] > array[j]) {
                let temp = array[i]
                array[i] = array[j]
                array[j] = temp
            }
        }
    }

    return array
}

let array = [1,4,2,4,21,0]
console.log(array)
let sortedArray = BubbleSort(array)
console.log(sortedArray)
